### PR TITLE
(chore) Update RealityCollective packages

### PR DESCRIPTION
**Pull request by SimonDarksideJ**

### CODE REVIEW

File: data/packages/com.realitycollective.service-framework.yml [hunk 1]
```diff
@@ -9,12 +9,7 @@ topics:
...
-gitTagPrefix: ''
-gitTagIgnore: '^0.{5,20}'
-minVersion: 1.0.0-preview.1
+minVersion: 1.0.0
 image: 'https://github.com/realitycollective/realitycollective.logo/raw/main/Branding/RealityCollectiveBanner_600x300.png'
-readme: 'development:README.md'
-readme_zhCN: ''
-displayName_zhCN: ''
-description_zhCN: ''
+readme: 'main:README.md'
```
Comment (favoyang) [inline data/packages/com.realitycollective.service-framework.yml:14]: Please keep the empty property: gitTagPrefix, gitTagIgnore, displayName_zhCN, description_zhCN.

**Review by favoyang — CHANGES_REQUESTED** [review]
Please check out the code review.

File: data/packages/com.realitycollective.service-framework.yml (hunk 1 above)
Comment (SimonDarksideJ) [reply data/packages/com.realitycollective.service-framework.yml]: Reaadded on request

**Review by favoyang — APPROVED** [review]

**Comment by SimonDarksideJ** [comment]
Thanks @favoyang for the quick turnaround :D

**Pull request merged**